master-firmware/src/priorities.h

```diff
@@ -17,6 +17,7 @@
 #define MAP_SERVER_PRIO (NORMALPRIO + 1)
 #define COLOR_SEQUENCE_SERVER_PRIO (NORMALPRIO)
 #define ARMS_CONTROLLER_PRIO (NORMALPRIO)
+#define MANIPULATOR_THREAD_PRIO (NORMALPRIO)
```

Comment:
Reading this file makes me think "Those guys have no idea how to allocate priorities" :D

Reply:
![i-dont-know-what-im-doing](https://user-images.githubusercontent.com/6131403/55502730-85df8680-564d-11e9-807f-2158e56d3979.jpg)